spec/controllers/aspect_memberships_controller_spec.rb

```diff
@@ -104,7 +106,7 @@
     it 'aspect membership does not exist' do
       delete :destroy, :format => :json, :id => 123
       expect(response).not_to be_success
-      expect(response.body).to include "Could not find the selected person in that aspect"
+      expect(response.body).to include I18n.t('aspect_memberships.destroy.no_membership')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.